Bottom: fae0ddc79a572f1d4b19900cba92b2b54d150374
Top:    27f6f8e16ca92774dbe54dca36f0b26d5a5be090
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 15:34:47 +0800

Refresh of autofs-5.0.7-allow-use-of-hosts-map-in-maps.patch

---

diff --git a/modules/mount_autofs.c b/modules/mount_autofs.c
index d33932c..319f2e4 100644
--- a/modules/mount_autofs.c
+++ b/modules/mount_autofs.c
@@ -216,8 +216,6 @@ int mount_mount(struct autofs_point *ap, const char *root, const char *name,
 	}
 	argv[argc] = NULL;
 
-	error(LOGOPT_ANY, "info type %s format %s map %s", info->type, info->format, info->map);
-
 	source = master_add_map_source(entry,
 				       info->type, info->format,
 				       time(NULL), argc, argv);
